src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -149,7 +131,36 @@ class CalculatorForm extends React.Component {
           ]}
           value={this.props.inputs.yellowRibbonRecipient}
           onChange={this.handleInputChange}/>
-        {amountInput}
+        { this.props.inputs.yellowRibbonRecipient === 'yes' ?
+          <div>
+            <Dropdown
+              label="Degree Level"
+              name={'yellowRibbonDegreeLevel'}
+              alt="Degree Level"
+              options={this.props.displayedInputs.yellowRibbonDegreeLevelOptions}
+              visible
+              value={this.props.displayedInputs.yellowRibbonDegreeLevel}
+              onChange={this.handleInputChange}/>
+            <Dropdown
+              label="Division or school"
+              name={'yellowRibbonDivision'}
+              alt="Division or school"
+              options={this.props.displayedInputs.yellowRibbonDivisionOptions}
+              visible
+              value={this.props.displayedInputs.yellowRibbonDivision}
+              onChange={this.handleInputChange}/>
+            <div>
+              <label htmlFor="yellowRibbonAmount">
+                Yellow Ribbon amount from school per year
+              </label>
+              <input
```

Comment:
I don't [think](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9365#issuecomment-396987338) this should be an input.